src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -49,6 +49,7 @@ public function getConfigTreeBuilder()
                         ->booleanNode('enabled')->defaultTrue()->info('To enable or disable eager loading')->end()
                         ->integerNode('max_joins')->defaultValue(30)->info('Max number of joined relations before EagerLoading throws a RuntimeException')->end()
                         ->booleanNode('force_eager')->defaultTrue()->info('Force join on every relation. If disabled, it will only join relations having the EAGER fetch mode.')->end()
+                        ->booleanNode('filter_relations')->defaultFalse()->info('Allow to filter the relation collection when the filter is on a nested property. By default, it will retrieve the full relation of the filtered resource. If enabled, only relations matching the filter will be returned.')->end()
```

Comment:
Only when filtering on a nested property because it only joins the result matching that filter.